app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
```

Comment:
- [ ] [REQUIRED] According to the ERD provided, there ought not to be foreign keys on the :likes and :comments associations. Kindly remove the foreign keys attached to the `has_many` :likes and :comments associations